src/PrestaShopBundle/Form/Admin/Sell/Product/Pricing/CatalogPriceRulesType.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Form\Admin\Sell\Product\Pricing;
+
+use PrestaShopBundle\Form\Admin\Type\IconButtonType;
+use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
+use PrestaShopBundle\Service\Routing\Router;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+
+class CatalogPriceRulesType extends TranslatorAwareType
+{
+    /**
+     * @var Router
+     */
+    private $router;
+
+    /**
+     * CatalogPriceRulesType constructor.
+     *
+     * @param TranslatorInterface $translator
+     * @param array<string> $locales
+     * @param Router $router
+     */
+    public function __construct(TranslatorInterface $translator, array $locales, Router $router)
+    {
+        parent::__construct($translator, $locales);
+        $this->router = $router;
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('manage_catalog_price_rules', IconButtonType::class, [
+                'label' => $this->trans('Manage catalog price rules', 'Admin.Catalog.Feature'),
+                'type' => 'link',
+                'attr' => [
+                    'class' => 'btn btn-link px-0 align-right',
+                    'href' => $this->router->generate('admin_catalog_price_rules_index'),
+                    'target' => '_blank',
+                ],
+                'icon' => 'open_in_new',
+            ])
+        ;
+    }
```

Comment:
This widget doesn't need any content, its special layout is handled via the form theme, however you can set all its options here as default options, it will simplify its integration in `PricingType`

```suggestion
    /**
     * {@inheritDoc}
     */
    public function configureOptions(OptionsResolver $resolver)
    {
        // Here as well you need to generate the legacy URL since catalog price rules has not been officially migrated yet, same principle as the URL in PricingType
        $catalogPriceRulesUrl = '';

        $resolver->setDefaults([
            'required' => false,
            'label' => $this->trans('Actiave catalog prices rules impacting the product price', 'Admin.Catalog.Feature'),
                'external_link' => [
                    'text' => $this->trans('[1]Manage catalog price rules[/1]', 'Admin.Catalog.Feature'),
                    'href' => $catalogPriceRulesUrl,
                ],
        ]);
    }
```